File: Data Structure II/Java/Day 2/HashMap.java
class MyHashMap {
    int[] map;
    public MyHashMap() {
        map=new int[10*10*10*10*10*10+1]; 
        Arrays.fill(map,-1);
    }
    
    public void put(int key, int value) {
        map[key]=value;
    }
    public int get(int key) {
        return map[key]; 
    }
    
    public void remove(int key) {
       map[key]=-1;
    }
}

/**
 * Your MyHashMap object will be instantiated and called as such:
 * MyHashMap obj = new MyHashMap();
 * obj.put(key,value);
 * int param_2 = obj.get(key);
 * obj.remove(key);
 */

File: Data Structure II/cpp/Day 2/HashMap.cpp
class MyHashMap {
public:
    int map[1000002];
    MyHashMap() {
        fill(map,map+1000001,-1);
    }
    
    void put(int key, int value) {
       map[key]=value; 
    }
    
    int get(int key) {
       return map[key]; 
    }
    
    void remove(int key) {
        map[key]=-1;
    }
};

/**
 * Your MyHashMap object will be instantiated and called as such:
 * MyHashMap* obj = new MyHashMap();
 * obj->put(key,value);
 * int param_2 = obj->get(key);
 * obj->remove(key);
 */